src/pods/home/home.container.tsx

```diff
@@ -10,110 +11,15 @@ export const HomeContainer = () => {
     const fullVerbCollection = await loadFullVerbCollection();
     verbsContext.setVerbCollection(fullVerbCollection);
 
-    // TODO: store this in local storage
-    verbsContext.setSelectedVerbs([
-      'be',
-      'bear',
-      'beat',
-      'become',
-      'begin',
-      'bite',
-      'blow',
-      'break',
-      'bring',
-      'build',
-      'burn',
-      'burst',
-      'buy',
-      'can',
-      'catch',
-      'choose',
-      'come',
-      'cost',
-      'cut',
-      'deal',
-      'dig',
-      'do',
-      'draw',
-      'dream',
-      'drink',
-      'drive',
-      'eat',
-      'fall',
-      'feed',
-      'fight',
-      'find',
-      'fly',
-      'forbid',
-      'forget',
-      'forgive',
-      'freeze',
-      'get',
-      'give',
-      'go',
-      'grow',
-      'hang',
-      'have',
-      'hear',
-      'hide',
-      'hit',
-      'hold',
-      'hurt',
-      'keep',
-      'know',
-      'lead',
-      'learn',
-      'leave',
-      'lend',
-      'let',
-      'light',
-      'lose',
-      'make',
-      'mean',
-      'meet',
-      'pay',
-      'put',
-      'read',
-      'ride',
-      'ring',
-      'rise',
-      'run',
-      'say',
-      'see',
-      'sell',
-      'send',
-      'set',
-      'sew',
-      'shake',
-      'shine',
-      'shoot',
-      'show',
-      'shut',
-      'sing',
-      'sit',
-      'sleep',
-      'smell',
-      'speak',
-      'spend',
-      'stand',
-      'steal',
-      'stick',
-      'sting',
-      'swear',
-      'sweep',
-      'swim',
-      'take',
-      'teach',
-      'tear',
-      'tell',
-      'think',
-      'throw',
-      'understand',
-      'wake',
-      'wear',
-      'win',
-      'write',
-    ]);
+    const selectedVerbs: string[] = JSON.parse(
+      localStorage.getItem('selectedVerbs')
+    );
+
+    if (selectedVerbs) verbsContext.setSelectedVerbs(selectedVerbs);
+    else {
+      verbsContext.setSelectedVerbs(initialSelected);
+      localStorage.setItem('selectedVerbs', JSON.stringify(initialSelected));
```

Comment:
Same here, mi advice

- Create a localStorage wrapper
- Use this wrapper in an "api" like file, and use promises, by using this approach we could easily migrate to rest-api without having to impact the rest of the application